examples/images_contours_and_fields/image_antialiasing.py

```diff
@@ -0,0 +1,85 @@
+"""
+==================
+Image Antialiasing
+==================
+
+Images are represented by discrete pixels, either on the screen or in an
+image file.  When data that makes up the image has a different resolution
+than its representation on the screen we will see aliasing effects.
+
+By default Matplotlib has a mild anti-aliasing filter turned on for images
+that are rendered to screen with either down sampling (i.e. the screen
+representation has fewer pixels than the data) or mild up-sampling, to a
+limit of three-times upsampling.
+
+If a different anti-aliasing filter is desired (or no filter), it can be
+specified in `.Axes.imshow` using the *interpolation* kwarg.
+"""
+
+import numpy as np
+import matplotlib.pyplot as plt
+
+###############################################################################
+# First we generate an image with varying frequency content:
+x = np.arange(500) / 500 - 0.5
+y = np.arange(500) / 500 - 0.5
+
+X, Y = np.meshgrid(x, y)
+R = np.sqrt(X**2 + Y**2)
+f0 = 10
+k = 250
+a = np.sin(np.pi * 2 * (f0 * R + k * R**2 / 2))
+
+
+###############################################################################
+# Now plot.  Note that these images are subsampled, with the images
+# representing 1000 data pixels in (approx) 604 image pixels.
+# The Moire patterns in the "nearest" interpolation are caused by the
+# high-frequency data being subsampled.  The "antialiased" image
+# still has some Moire patterns as well, but they are greatly reduced.
+# We can reduce them more, as in the examples below, but better filters
```

Comment:
The 'better filter + compultationally expensive' information should be moved to the section where it's actually rendered. I'd remove this sentence here completely (If you really want something here, it would just be "See below for even better filters.").

Reply:
Thanks @timhoffm made the suggested changes as noted above.